packages/server/graphql/CompiledQueryCache.ts

```diff
@@ -45,7 +44,7 @@ export default class CompiledQueryCache {
     if (compiledQuery) return compiledQuery
     const r = await getRethink()
     let queryString = await r.table('QueryMap').get(docId)('query').default(null).run()
-    if (!queryString && !PROD) {
+    if (!queryString && !__PRODUCTION__) {
```

Comment:
Verified working. Here's what it looks like when we run `yarn build`. No reference to `queryMap.json` in prod 🎉 

<img width="795" alt="Screenshot 2023-09-06 at 3 27 15 PM" src="https://github.com/ParabolInc/parabol/assets/5514175/4da3a24f-3fd6-4acc-bdcc-2b5107c88f17">